### PR TITLE
improve efficiency && add emo_isPureEmojiString method

### PR DESCRIPTION
**Pull request by wujichao**

### CODE REVIEW

File: SearchEmojiOnString-iOS/NSString+EMOEmoji.m [hunk 1]
```diff
@@ -165,11 +165,103 @@ - (BOOL)emo_containsEmoji
...
+                      hs == 0x3030 ||
+                      hs == 0x2b55 ||
+                      hs == 0x2b1c ||
+                      hs == 0x2b1b ||
+                      hs == 0x2b50)
+             {
+                 containsEmoji = YES;
+             }
+         }
+         if (!containsEmoji) {
```
Comment (GabrielMassana) [inline SearchEmojiOnString-iOS/NSString+EMOEmoji.m:258]: can you add an extra line between } and if (!containsEmoji) { ??

File: SearchEmojiOnString-iOS/NSString+EMOEmoji.m [hunk 2]
```diff
@@ -165,11 +165,103 @@ - (BOOL)emo_containsEmoji
...
+             isPureEmojiString = NO;
+             *stop = YES;
+         }
+     }];
+    return isPureEmojiString;
```
Comment (GabrielMassana) [inline SearchEmojiOnString-iOS/NSString+EMOEmoji.m:264]: Can you add an extra line between }]; and return isPureEmojiString;

**Comment by GabrielMassana** [comment]
Hello,

Thanks for the PR!

I like your approach!

Are you confident adding some small Tests? If not, I can do it. 

I also added 2 small comments to add 2 extra lines.

Cheers

**Comment by wujichao** [comment]
Hi,
I can't make it pass CI with `iphonesimulator8.1`, could you help me?

**Comment by GabrielMassana** [comment]
I'll merge and check . 

Thx

**Comment by GabrielMassana** [comment]
I think there is a problem in xctool or in the travis configuration after the new xcode version was released.

I'm using now xcodebuild  insted of xctool for the CI, and works. :)

**Comment by wujichao** [comment]
great, thanks

**Pull request merged**